spec/models/validations/household_validations_spec.rb

```diff
@@ -215,6 +215,24 @@
           .to include(match(I18n.t("validations.household.prevten.la_general_needs.internal_transfer")))
       end
     end
+
+    context "when referral is nominated by a local housing authority" do
+      it "cannot have `other landlord`" do
```

Comment:
Any reason to use backticks over `'` here?

Reply:
No, just a habit 👀 Also, looks a bit more distinguishable within the sentence, cause otherwise we'd end up with ''' at the end. I'm not married to the idea, how important is it to you?

Reply:
Not very. I was just curious.